gtsfm/evaluation/metrics_report.py

```diff
@@ -42,7 +42,31 @@ def create_table_for_scalar_metrics(metrics_dict: Dict[str, Union[float, int]])
     Returns:
         Table with scalar metrics and their values in HTML format.
     """
-    table = {"Metric name": list(metrics_dict.keys()), "Value": list(metrics_dict.values())}
+    table = {
+        "Metric name": list(metrics_dict.keys()),
+        "Value": list(metrics_dict.values()),
+    }
+    return tabulate(table, headers="keys", tablefmt="html")
```

Comment:
probably needs to be reformatted with the 120 character limit?

Reply:
Reverted.